recipes/xmscore/meta.yaml

```diff
@@ -0,0 +1,51 @@
+{% set version = "3.0.7" %}
+
+package:
+  name: xmscore
+  version: {{ version }}
+
+source:
+  url: https://github.com/Aquaveo/xmscore/archive/{{ version }}.tar.gz
+  sha256: 98acd332ac5a14602539e218db82b2970dbeed2b149e0d47dcbd80c6778801c7
+
+build:
+  number: 0
+  skip: True  # [not py36]
```

Comment:
Skips all but Python 3.6. We need that but I doubt the limitation upstream is true. There are very few reason for a 3.6 package to not work with 3.7.